src/main/java/com/mael/ttt/players/Player.java

```diff
@@ -7,4 +7,5 @@
 
     int getMove(Board board);
     Mark getMark();
+    PlayerType getType();
```

Comment:
This seems odd.
Why is the method `getType()` needed?
This indicates that there's either something wrong with the abstraction of a `Player` itself, or that a type-check violates the idea behind polymorphism.

Also it seems that this method is only used in the tests for players.